docs/manual-CN/statement-syntax/data-administration-statements/download-and-ingest-statements/download-syntax.md

```diff
@@ -0,0 +1,13 @@
+Nebula 通过 `DOWNLOAD` 语句从 `HDFS` 下载SST文件到download目录。
+
+请确认 `Hadoop` 已安装，且 `HADOOP_HOME` 已配置
+
+```
+echo $HADOOP_HOME
+```
+
+### 数据下载
+
+```
+DOWNLOAD HDFS "hdfs://${HADOOP_HOST}:${HADOOP_PORT}/${HADOOP_PATH}"
```

Comment:
add `nebula>` indicates that the command is executed through the console.

Reply:
emmm... This the language syntax not an example should not running in console.